NEWS.md

```diff
@@ -111,10 +111,6 @@ based on [Common Terminology Criteria for Adverse Events (CTCAE) v5.0](https://c
 
 - New vignette "Hy's Law Implementation" (#1637)
 
-- `ADLB` metadata data set called `atoxgr_criteria_ctcv5` updated to remove unit check for
-`HYPERURICEMIA` as grade criteria based on `ANRHI` only.  This metadata holds criteria for lab grading
-based on [Common Terminology Criteria for Adverse Events (CTCAE) v5.0](https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm) (#1650)
-
```

Comment:
This news issue is driving me crazy!

Reply:
Yes, it's a zombie item. You can't get rid of it. Each time you merge devel into your feature branch it pops up elsewhere.